pages/services-mobile-development-android.js

```diff
@@ -1,55 +1,91 @@
 import ContentNavigation from '../components/content/navigation';
 
-import { servicesMobileDevelopment } from '../main.config';
+import { servicesMobileDevelopment, works } from '../main.config';
+
+import Works from '../components/portfolio/works';
+import CallToActionButton from '../components/content/CallToActionButton';
+import CalendlyIframe from '../components/content/CalendlyIframe';
+import FeaturesList from '../components/content/FeaturesList';
+
+const whyAndroid = [
+  {
+    icon: '/static/services/world.svg',
+    alt: 'world',
+    description: <p>Android market is growing at a rapid pace, and you can become a part of it. At the end of 2018, <a href="//www.statista.com/statistics/266136/global-market-share-held-by-smartphone-operating-systems/" target="_blank" rel="noopener noreferrer nofollow">88% of all smartphones</a> sold to end users were using Android. Your business will go global.</p>,
+  },
+  {
+    icon: '/static/services/audience.svg',
+    alt: 'audience',
+    description: <p>Android devices are more affordable and accessible than their Apple counterparts due to a wide range of prices that Android manufacturers offer. Therefore, your app can <b>reach any audience</b> you want.</p>,
+  },
+  {
+    icon: '/static/services/business-and-finance.svg',
+    alt: 'business and finance',
+    description: <p>Android-based IoT market is growing at an unprecedented rate, so do not hesitate to enter this <b>profitable niche</b> while it is not yet overcrowded.</p>,
+  },
+  {
+    icon: '/static/services/compatible-with-multiple-platforms.svg',
+    alt: 'compatible with multiple platforms',
+    description: <p>Android is <b>compatible</b> with multiple devices, including smart TVs, watches, bracelets, tablets, and even vacuum cleaners.</p>,
+  },
+  {
+    icon: '/static/services/fastest-technology.svg',
+    alt: 'fastest technology',
+    description: <p>Android development process is <b>fast and easy</b> due to the huge open-source communities. One can also dig into the source code of any app to see how it works.</p>,
+  },
+  {
+    icon: '/static/services/google-play.svg',
+    alt: 'google play',
+    description: <p>With Android, you can <b>publish your app as soon as it is finished</b>. You have fewer guidelines to follow and avoid going through a lengthy process of app approval.</p>,
+  },
+  {
+    icon: '/static/services/light-bulb.svg',
+    alt: 'light bulb',
+    description: <p>Due to the absence of narrow guidelines, Android developers have <b>full freedom</b> and can create and deploy almost any product you may think of.</p>,
+  },
+  {
+    icon: '/static/services/working-impeccably.svg',
+    alt: 'working impeccably',
+    description: <p>The newest Android devices have <b>hardware so powerful</b> that your app can do whatever you want it to.</p>,
+  },
+];
+
+const androidWorks = works.filter(({ title }) => {
+  return ['Pree', 'ParkQX', 'Tetrami'].includes(title);
+});
```

Comment:
@soloviovmax , could it be oneliner? Like this: `const androidWorks = works.filter(({ title }) => ['Pree', 'ParkQX', 'Tetrami'].includes(title));`

Reply:
Done